project_recalculate/wizard/recalculate_wizard.py

```diff
@@ -0,0 +1,60 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright :
+#        (c) 2014 Antiun Ingenieria S.L. (Madrid, Spain, http://www.antiun.com)
+#                 Endika Iglesias <endikaig@antiun.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+from openerp.exceptions import Warning  # , RedirectWarning
+from openerp.tools.translate import _
+
+
+class ProjectRecalculateWizard(models.TransientModel):
+    _name = 'project.recalculate.wizard'
+
+    project = fields.Many2one(comodel_name='project.project', readonly=True)
+    project_name = fields.Char(related='project.name', readonly=True)
```

Comment:
I can't get the name with project i try put this:
<field string="Project name" name="project.name"/>
and when i update addon "the field 'name' don't exist